<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rendv</class>
 <widget class="QDialog" name="Rendv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QClient{
	background-color: #071e26;
}
QWidget {	
	border:1px solid #dfdfdf;
	border-radius: 5px;
	background-color: #ffffff;
}

QPushButton{
	
	border-raduis:5px
}
QPushButton:pressed {
	background-color: rgb(189, 189, 189);
}
#side_menu{
	border-radius: 20px;
	background-color: rgb(0,0,0);
}
#top_widget{

	background-color: rgb(0,0,0);
}
#form_2{
	
	
	background-color: rgb(255, 255, 255);
border-top-left-radius: 50px;
}

#widget_2 QLineEdit{
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgb(0,0,0);
padding-bottom:7px

}
#widget_2{
	
	/* background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);*/
	/*background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4d4d4d, stop: 0.1 #646464, stop: 1 #5d5d5d);*/
   background-color:QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4aafdb, stop: 1 #090763);
}
#options_2{
background-color: rgb(0,0,0);
border-bottom-right-radius: 50px;

}
#widget_2 #recherche_2{
	
	background-color: rgb(204, 204, 204);
	border:1px solid rgb(0,0,0);
}

#ajouter{
 color: rgb(0,0,0);
    background-color: rgb(181, 181, 181);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}


#modifier{
color: rgb(0,0,0);
    background-color: rgb(181, 181, 181);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}

#supprimer{
 color: rgb(0,0,0);
    background-color: rgb(181, 181, 181);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;
}

#ajouter:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

#modifier:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}

#supprimer:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
}
#pdf_3{
	
	background-color: #071e26;
border:0px ;
}
#stat_3{
	
	background-color: #071e26;
border:0px ;
}
#widget QLineEdit{
background-color: rgb(204, 204, 204);
border:1px solid rgb(0,0,0);

}
#widget
{
	border-radius: 20px;
	background-color: rgb(0,0,0);
}
</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>1641</width>
     <height>831</height>
    </rect>
   </property>
   <widget class="QLabel" name="form_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>371</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="options_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>161</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouter">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifier">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>supprimer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Identifiant de l'Employ√©</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>981</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-raduis:5px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>961</width>
      <height>91</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="recherche_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>921</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>  recherche</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ID_RDV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DATE_RDV</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ASC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DESC</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Identifiant du Client</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1641</width>
      <height>841</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="style.qrc">:/new/prefix1/icons/blue2.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Identifiant du Rendez-vous</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>590</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>07:10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>09:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15:00</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="imp_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>540</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);
    background-color: rgb(185, 185, 185);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;</string>
    </property>
    <property name="text">
     <string>trie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imp">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);
    background-color: rgb(185, 185, 185);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;</string>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="timeSpec">
     <enum>Qt::LocalTime</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>670</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>CONTROLER VOS LUMI√àRES</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CONTROLER VOTRE VITRAGE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CONTROLER VOTRE MOTEUR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CONTROLER VOTRE FREINAGE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CONTROLER VOTRE SUSPENSION ET VOS PNEUS</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="imp_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>630</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0);
    background-color: rgb(185, 185, 185);
    border-width: 1px;
    border-color: #1e1e1e;
    border-style: solid;
    border-radius: 30px;
    padding: 3px;
    font-size: 12px;
    padding-left: 5px;
    padding-right: 5px;</string>
    </property>
    <property name="text">
     <string>Graphe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email@gmail.com</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>391</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>530</y>
      <width>561</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>1071</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-raduis:5px;</string>
    </property>
   </widget>
   <zorder>tableView_3</zorder>
   <zorder>label</zorder>
   <zorder>form_2</zorder>
   <zorder>options_2</zorder>
   <zorder>ajouter</zorder>
   <zorder>modifier</zorder>
   <zorder>supprimer</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>tableView_2</zorder>
   <zorder>widget</zorder>
   <zorder>lineEdit_5</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>imp_2</zorder>
   <zorder>imp</zorder>
   <zorder>dateEdit</zorder>
   <zorder>comboBox_3</zorder>
   <zorder>imp_3</zorder>
   <zorder>lineEdit_6</zorder>
   <zorder>widget_3</zorder>
   <zorder>calendarWidget</zorder>
  </widget>
  <widget class="QWidget" name="top_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1851</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1780</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 160px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/retour2.png</normaloff>:/new/prefix1/icons/retour2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Monotype Corsiva&quot;;</string>
    </property>
    <property name="text">
     <string>Etat du Rendez-vous</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/etat.png</normaloff>:/new/prefix1/icons/etat.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_excel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: italic 14pt &quot;Monotype Corsiva&quot;;</string>
    </property>
    <property name="text">
     <string>    Excel</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/excel.png</normaloff>:/new/prefix1/icons/excel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;MV Boli&quot;;
border-radius:20px;</string>
    </property>
    <property name="text">
     <string>gestion des rendez-vous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/rend.png</normaloff>:/new/prefix1/icons/rend.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: italic 14pt &quot;Monotype Corsiva&quot;;</string>
    </property>
    <property name="text">
     <string>votre compte</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/emp.png</normaloff>:/new/prefix1/icons/emp.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="side_menu" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>191</width>
     <height>831</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipements</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/equi.png</normaloff>:/new/prefix1/icons/equi.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>         Clients</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/cli.png</normaloff>:/new/prefix1/icons/cli.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>        Vehicules</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/icons/vehi.png</normaloff>:/new/prefix1/icons/vehi.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>top_widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>side_menu</zorder>
 </widget>
 <resources>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
